components/scream/src/dynamics/homme/atmosphere_dynamics.cpp

```diff
@@ -371,21 +368,12 @@ void HommeDynamics::initialize_impl (const RunType run_type)
     m_d2p_remapper->registration_begins();
 
     // ftype==FORCING_0:
-    //  1) remap Q_rgn->Q_dyn
-    //  2) compute FQ_dyn=(Q_dyn-Q_dyn_old)/dt
+    //  1) remap Q_pgn->FQ_dyn
+    //  2) compute FQ_dyn=dp3d*(FQ_dyn-Q_dyn_old)/dt
     // ftype!=FORCING_0:
-    //  1) compute FQ_rgn=Q_rgn-Q_rgn_old
-    //  2) remap FQ_rgn->FQ_dyn
-    //  3) Q_dyn=Q_dyn_old+FQ_dyn
-    if (params.ftype==Homme::ForcingAlg::FORCING_2) {
-      // Need a tmp for dQ_phys
-      using namespace ShortFieldTagsNames;
-      create_helper_field("FQ_phys",{COL,CMP,LEV},{ncols,qsize,nlevs},pgn);
-      m_p2d_remapper->register_field(m_helper_fields.at("FQ_phys"),m_helper_fields.at("FQ_dyn"));
-    } else {
-      // Can remap Q directly into FQ, tendency computed in pre_process step
-      m_p2d_remapper->register_field(*get_group_out("Q",pgn).m_bundle,m_helper_fields.at("FQ_dyn"));
-    }
+    //  1) remap Q_pgn->FQ_dyn
+    // Remap Q directly into FQ, tendency computed in pre_process step
```

Comment:
Where is the tendency computed though?

Reply:
See comment below tagging the spot in the code where the ftype=0 tendency is computed (old code).